src/Tribe/Attendees_Table.php

```diff
@@ -602,7 +595,7 @@ public function extra_tablenav( $which ) {
 		$nav = [
 			'left' => [
 				'print'  => sprintf( '<input type="button" name="print" class="print button action" value="%s">', esc_attr__( 'Print', 'event-tickets' ) ),
-				'export' => sprintf( '<a target="_blank" href="%s" class="export button action">%s</a>', esc_url( $export_url ), esc_html__( 'Export', 'event-tickets' ) ),
+				'export' => sprintf( '<a target="_blank" href="%s" class="export button action">%s</a>', esc_url( tribe( 'tickets.attendees' )->get_export_url() ), esc_html__( 'Export', 'event-tickets' ) ),
```

Comment:
For security issues we must have `rel="noopener noreferrer"` when the `target` is `_blank`